pkgs/development/python-modules/nipype/default.nix

```diff
@@ -66,18 +71,34 @@ buildPythonPackage rec {
     scipy
     simplejson
     traits
+    neurdflib
     xvfbwrapper
   ] ++ stdenv.lib.optional (!isPy3k) [
     configparser
     futures
   ];
 
-  checkInputs = [ pytest mock pytestcov pytest_xdist pytest-forked codecov which glibcLocales ];
+  buildInputs = [ 
```

Comment:
These are typically *only* checkInputs

Reply:
Here they say they want users to be able to run the tests:
https://github.com/nipy/nipype/pull/2649#issuecomment-407821835
So, strictly speaking, it's not a requirement to use the library. Therefore, I am of the opinion we should patch `pytest*` out of `REQUIRES`.

Reply:
I will do that, however they seem to need a testing dep at runtime somehow https://github.com/nipy/nipype/issues/2680